packages/directory/tests/dummy/mirage/serializers/application.js

```diff
@@ -1,4 +1,5 @@
 import { JSONAPISerializer } from 'ember-cli-mirage';
 
 export default JSONAPISerializer.extend({
+  alwaysIncludeLinkageData: true,
```

Comment:
Can we comment on why this is needed? and remove the trailing `,`

Reply:
It's not needed anymore since I reverted the mirage version upgrade. Will remove